### ISSUE TITLE
Tool to Replace GCU Stencil Text

### ISSUE THREAD

**Issue by pheller** [ProdigyReloaded/delivery-system#53]
In order to programmatically create content, a tool that can replace the stencil text placeholder in a GCU recipe is needed.  The format of a GCU recipe file is as follows:

Prologue:
```
<< "GCULIST", 0x50020001, header_len::16-little >>
``` 

Header:
```
<<
  0x01011500, recipe_name::binary-size(21),
  0x02011f00, recipe_descr::binary-size(31),
  0x03011b00, object_id::binary-size(27),
  0x04010300, partition_num:: binary-size(3),
  0x05010200, list_count::16-little,
  0x0601, list_names_len::16-little, list_names::binary-size(list_names_len)
>>
```

Each list:

```
  0x0200, list_size::16-little,
  0x01020b00, list_name::binary-size(11),
  0x02020400, field_no::binary-size(4),
  0x03020600, type, display, edit,  (see below)
  0x04020800, 0::binary-size(8),
  0x05020400, 0::binary-size(4),
  0x06020400, 0::binary-size(4),
  0x09021d00, 0::binary-size(29),
  0x0a021d00, 0::binary-size(29),
  0x0702, stencil_size::16-little, stencil_naplps::binary-size(stencil_size),
  0x0b02, original_size::16-little, original_naplps::binary-size(original_size)
>>
```

A tool to merge stencil text should be able to fill multiple stencil fields in a single pass, and the manner this is done is up to the developer, though it would be convenient if the text could be given as command line arguments.

Basically the text needs to replace the placeholder text, and the size of the field recalculated, then the updated GCU LIST saved.  Subsequently, `cook.exe` is run to process the GCU LIST into a page element object with embedded presentation display object, and fields as may be specified in the GCULIST.
